### DISCUSSION TITLE
What’s the “Platform Blue” mentioned in the Readme?

### DISCUSSION THREAD

**Discussion by chexo3** [jrouwe/JoltPhysics#362] [Q&A]
Never heard of that before. Since it sounds like it’s under NDA I’m not sure how much you can divulge but I am curious.

**Comment by jrouwe** [comment]
I thought it was pretty obvious... Take the AAA game mentioned on the front page. What platforms did it come out on?

**Reply by chexo3** [reply]
Ah, PlayStation?

**Reply by Admer456** [reply]
It appears to be PlayStation. I had no idea this whole time, TIL

**Comment by jankrassnigg** [comment]
Intel PCs obviously. :-P

Though I always found it funny that even after release, platforms are often not mentioned by their real name. I worked on the Vision engine back when it still existed and it was full of #ifdefs with platform code names, that most people had already forgotten about. (Do you remember which platform was code named Xenon?)

**Reply by jrouwe** [reply]
Yes, I remember a lot of ifdefs like that (even with code names for code names)! Seems to be public knowledge now that Xenon was the X360.
